src/main/scss/styles.scss

```diff
@@ -1,6 +1,4 @@
-$primary-font: "Helvetica Neue", "Open Sans", sans-serif;
-$background-color: #0E0B16;
-$text-color: #FFF;
+@import '_variables';
```

Comment:
No need to include the underscore when [importing](http://sass-lang.com/guide#topic-5).